src/containers/my-cooperations/cooperations-container/CooperationContainer.constants.tsx

```diff
@@ -1,28 +1,38 @@
-import Box from '@mui/material/Box'
-import StatusChip from '~/components/status-chip/StatusChip'
-import AppChip from '~/components/app-chip/AppChip'
 import { getFormatedDate } from '~/utils/helper-functions'
+import StatusChip from '~/components/status-chip/StatusChip'
+import UserProfileInfo from '~/components/user-profile-info/UserProfileInfo'
+import SubjectLevelChips from '~/components/subject-level-chips/SubjectLevelChips'
 import { Cooperation } from '~/types'
 import { styles } from '~/containers/my-cooperations/cooperations-container/CooperationContainer.styles'
+import { Typography } from '@mui/material'
```

Comment:
Move above and do it like this `import Typography from '@mui/material'/Typography `

Reply:
fixed